### ISSUE TITLE
Assertion Error - Feature 'MEMBER_PROFILES' is invalid.

### ISSUE THREAD

**Issue by matthewre** [JaxkDev/DiscordBot#128]
```
[2024-04-27T16:41:52.603687+00:00] InternalThread.CRITICAL: AssertionError: "Feature 'MEMBER_PROFILES' is invalid." (EXCEPTION) in "phar:///home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/src/Models/Guild/Guild.php" at line 400 [] []
[2024-04-27T16:41:52.604121+00:00] InternalThread.CRITICAL: #0 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/src/Models/Guild/Guild(222): JaxkDev\DiscordBot\Models\Guild\Guild->setFeatures(array[12]) [] []
[2024-04-27T16:41:52.604153+00:00] InternalThread.CRITICAL: #1 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/src/InternalBot/ModelConverter(376): JaxkDev\DiscordBot\Models\Guild\Guild->__construct(string[19] 1231274627452502146, string[12] Velo Network, string[32] 403c0152c1270af310ab2047152e6ff9, NULL , NULL , string[18] 592426234194165772, NULL , integer 300, NULL , NULL , object JaxkDev\DiscordBot\Models\Guild\VerificationLevel#1274, object JaxkDev\DiscordBot\Models\Guild\DefaultMessageNotificationLevel#1320, object JaxkDev\DiscordBot\Models\Guild\ExplicitContentFilterLevel#549, array[0], array[12], object JaxkDev\DiscordBot\Models\Guild\MfaLevel#402, NULL , string[19] 1231311561117925501, integer 13, string[19] 1231276514910273686, NULL , integer 500000, string[7] velopvp, NULL , string[32] 9116490ecb79fd9e8a0d5460a2f60fdf, object JaxkDev\DiscordBot\Models\Guild\PremiumTier#408, integer 16, string[5] en-US, string[19] 1231276408001527860, integer 25, integer 360, object JaxkDev\DiscordBot\Models\Guild\NsfwLevel#1302, array[0], false, NULL ) [] []
[2024-04-27T16:41:52.604170+00:00] InternalThread.CRITICAL: #2 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/src/InternalBot/Handlers/DiscordEventHandler(316): JaxkDev\DiscordBot\InternalBot\ModelConverter::genModelGuild(object Discord\Parts\Guild\Guild#401) [] []
[2024-04-27T16:41:52.604179+00:00] InternalThread.CRITICAL: #3 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/evenement/evenement/src/EventEmitterTrait(143): JaxkDev\DiscordBot\InternalBot\Handlers\DiscordEventHandler->onGuildJoin(object Discord\Parts\Guild\Guild#401, object JaxkDev\DiscordBot\InternalBot\Discord#21, NULL ) [] []
[2024-04-27T16:41:52.604199+00:00] InternalThread.CRITICAL: #4 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/team-reflex/discord-php/src/Discord/Discord(784): Discord\Discord->emit(string[12] GUILD_CREATE, array[3]) [] []
[2024-04-27T16:41:52.604209+00:00] InternalThread.CRITICAL: #5 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/promise/src/FulfilledPromise(42): Discord\Discord->Discord\{closure}(object Discord\Parts\Guild\Guild#401) [] []
[2024-04-27T16:41:52.604218+00:00] InternalThread.CRITICAL: #6 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/promise/src/Promise(66): React\Promise\FulfilledPromise->done(object Closure#555, object Closure#560) [] []
[2024-04-27T16:41:52.604225+00:00] InternalThread.CRITICAL: #7 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/promise/src/Promise(168): React\Promise\Promise::React\Promise\{closure}(object React\Promise\FulfilledPromise#410) [] []
[2024-04-27T16:41:52.604232+00:00] InternalThread.CRITICAL: #8 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/promise/src/Promise(231): React\Promise\Promise->settle(object React\Promise\FulfilledPromise#410) [] []
[2024-04-27T16:41:52.604239+00:00] InternalThread.CRITICAL: #9 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/promise/src/Deferred(36): React\Promise\Promise::React\Promise\{closure}(object Discord\Parts\Guild\Guild#401) [] []
[2024-04-27T16:41:52.604246+00:00] InternalThread.CRITICAL: #10 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/promise/src/FulfilledPromise(42): React\Promise\Deferred->resolve(object Discord\Parts\Guild\Guild#401) [] []
[2024-04-27T16:41:52.604252+00:00] InternalThread.CRITICAL: #11 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/promise/src/Promise(61): React\Promise\FulfilledPromise->done(array[2], array[2], NULL ) [] []
[2024-04-27T16:41:52.604259+00:00] InternalThread.CRITICAL: #12 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/team-reflex/discord-php/src/Discord/Discord(817): React\Promise\Promise->done(array[2], array[2]) [] []
[2024-04-27T16:41:52.604266+00:00] InternalThread.CRITICAL: #13 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/team-reflex/discord-php/src/Discord/Discord(667): Discord\Discord->handleDispatch(object stdClass#175) [] []
[2024-04-27T16:41:52.604273+00:00] InternalThread.CRITICAL: #14 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/team-reflex/discord-php/src/Discord/Discord(633): Discord\Discord->processWsMessage(object stdClass#175) [] []
[2024-04-27T16:41:52.604279+00:00] InternalThread.CRITICAL: #15 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/evenement/evenement/src/EventEmitterTrait(143): Discord\Discord->handleWsMessage(object Ratchet\RFC6455\Messaging\Message#499, object Ratchet\Client\WebSocket#307) [] []
[2024-04-27T16:41:52.604287+00:00] InternalThread.CRITICAL: #16 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/ratchet/pawl/src/WebSocket(72): Ratchet\Client\WebSocket->emit(string[7] message, array[2]) [] []
[2024-04-27T16:41:52.604298+00:00] InternalThread.CRITICAL: #17 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/ratchet/rfc6455/src/Messaging/MessageBuffer(248): Ratchet\Client\WebSocket->Ratchet\Client\{closure}(object Ratchet\RFC6455\Messaging\Message#499, object Ratchet\RFC6455\Messaging\MessageBuffer#212) [] []
[2024-04-27T16:41:52.604305+00:00] InternalThread.CRITICAL: #18 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/ratchet/rfc6455/src/Messaging/MessageBuffer(194): Ratchet\RFC6455\Messaging\MessageBuffer->processData(string[6931] .~...]I.\.u.+.^&b..C.LRm..8.M.6......kh..f...'."@...x.l..:@ g..........;..zC.W.i) [] []
[2024-04-27T16:41:52.604319+00:00] InternalThread.CRITICAL: #19 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/evenement/evenement/src/EventEmitterTrait(143): Ratchet\RFC6455\Messaging\MessageBuffer->onData(string[6931] .~...]I.\.u.+.^&b..C.LRm..8.M.6......kh..f...'."@...x.l..:@ g..........;..zC.W.i) [] []
[2024-04-27T16:41:52.604327+00:00] InternalThread.CRITICAL: #20 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/stream/src/Util(71): Evenement\EventEmitter->emit(string[4] data, array[1]) [] []
[2024-04-27T16:41:52.604334+00:00] InternalThread.CRITICAL: #21 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/evenement/evenement/src/EventEmitterTrait(143): React\Stream\Util::React\Stream\{closure}(string[2707] ....M......&(.YF'...F{.X0G:.....Q.2......d...h."..7..!.m.*l..I.....x.Nj.... I..1) [] []
[2024-04-27T16:41:52.604341+00:00] InternalThread.CRITICAL: #22 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/stream/src/DuplexResourceStream(196): Evenement\EventEmitter->emit(string[4] data, array[1]) [] []
[2024-04-27T16:41:52.604349+00:00] InternalThread.CRITICAL: #23 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/event-loop/src/StreamSelectLoop(246): React\Stream\DuplexResourceStream->handleData(resource Resource id #335) [] []
[2024-04-27T16:41:52.604362+00:00] InternalThread.CRITICAL: #24 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/react/event-loop/src/StreamSelectLoop(213): React\EventLoop\StreamSelectLoop->waitForStreamActivity(integer 37145) [] []
[2024-04-27T16:41:52.604370+00:00] InternalThread.CRITICAL: #25 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/vendor/team-reflex/discord-php/src/Discord/Discord(1496): React\EventLoop\StreamSelectLoop->run() [] []
[2024-04-27T16:41:52.604377+00:00] InternalThread.CRITICAL: #26 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/src/InternalBot/Client(149): Discord\Discord->run() [] []
[2024-04-27T16:41:52.604385+00:00] InternalThread.CRITICAL: #27 /home/minecraft/multicraft/servers/server713167/plugins/DiscordBot (2).phar/src/Communication/Thread(52): JaxkDev\DiscordBot\InternalBot\Client->__construct(object JaxkDev\DiscordBot\Communication\Thread#3) [] []
[2024-04-27T16:41:52.604392+00:00] InternalThread.CRITICAL: #28 (): JaxkDev\DiscordBot\Communication\Thread->run() [] []```

**Comment by JaxkDev** [comment]
The reason I disallowed this is due to the discord documentation as not having it listed as not all features are on the guild list.
https://discord.com/developers/docs/resources/guild#guild-object-guild-features

**Comment by JaxkDev** [comment]
This was fixed in v3.0.3 as per the changelogs.
Update your plugin.
